docs/prd.md

```
@@ -0,0 +1,71 @@
+- 5 Elements (Pancha Mahabhuta), Pancha Kosha, Three Doshas, Food qualities & Gunas
+
+### 5. Recipes (`4_recipes.md`)
+- Seasonal Ayurvedic recipes
+- Anti-allergic soup, sprouts, wild garlic pesto, flu-bomb decoction
+
+### 6. Consultation (`5_consultation.md`)
+- Service offerings (in-person Berlin + online)
+- Consultation process description
+- Dosha-specific session types (Vata, Pitta, Kapha)
+
+### 7. Contact (`6_contact.md`)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)